tests/live/test_server.py

```diff
@@ -0,0 +1,48 @@
+import uuid
+
+from mixcoatl.resource import Endpoint
+from mixcoatl.infrastructure.server import Server
+from mixcoatl.admin.job import Job
+
+
+class TestServerLive:
+    def setup(self):
+        # load an endpoint that this file must be hosted outside of git
+        # because it contains credentials
+        self.endpoint = Endpoint.multiple_from_file("../../../secret-test-data/endpoints.json")['saas']
+
+    def test_saas_server_create(self):
+        """Test if we can create a VM using an live endpoint object and shut it down"""
+
+        # This cli command would make a server similar to below
+        # dcm-create-server --machineimage 29710 --datacenter 1273 \
+        #     --name igable-terminateme --productid t1.micro --budgetid 1512 --description longstring
+
+        self.server = Server(endpoint=self.endpoint)
+        self.server.set_from_file("../../../secret-test-data/ubuntu-trusty-14.04-amd64-server-20150629.json")
```

Comment:
Sounds like a good idea. Were you somehow able to trigger this test without specifically calling it?